# PUMP_CONTROLLER message data

Header header

float32 pump_speed_sp
float32 spray_speed_sp